### PR TITLE
Create merge_sort_fastest.py

### PR DESCRIPTION
**Pull request by yesIamHasi**
Python implementation of merge sort algorithm.
Takes an average of 0.6 microseconds to sort a list of length 1000 items.
Best Case Scenario : O(n)
Worst Case Scenario : O(n)

### CODE REVIEW

File: sorts/merge_sort_fastest.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    b = LIST[-1]
+    while (LIST.index(a) == LIST.index(b) and len(LIST) <=2):
+        a = min(LIST)
+        b = max(LIST)
+        start.append(a)
+        end.append(b)
+        LIST.remove(a)
+        LIST.remove(b)
+    end.reverse()
+return start + end
```
Comment (christianbender) [inline sorts/merge_sort_fastest.py]: You must indent this line.

File: sorts/merge_sort_fastest.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+'''
+Python implementation of merge sort algorithm.
+Takes an average of 0.6 microseconds to sort a list of length 1000 items.
+Best Case Scenario : O(n)
+Worst Case Scenario : O(n)
+'''
```
Comment (christianbender) [inline sorts/merge_sort_fastest.py]: Your function don't work. I tested it:
```python
def merge_sort(LIST):
    start = []
    end = []
    a = LIST[0]
    b = LIST[-1]
    while (LIST.index(a) == LIST.index(b) and len(LIST) <=2):
        a = min(LIST)
        b = max(LIST)
        start.append(a)
        end.append(b)
        LIST.remove(a)
        LIST.remove(b)
    end.reverse()
    return (start + end)

liste = [4,1,99,3,2,2,0,100]
print liste
print merge_sort(liste)
```  

The output is:  
> [4, 1, 99, 3, 2, 2, 0, 100]
[]

**Comment by harshildarji** [comment]
@christianbender I have modified the code a little to make it work as (_contributor_) expected! It is working on my side, but it will be good if you also confirm it!

> code:
```python
def merge_sort(LIST):
    start = []
    end = []
    while len(LIST) > 1:
        a = min(LIST)
        b = max(LIST)
        start.append(a)
        end.append(b)
        LIST.remove(a)
        LIST.remove(b)
    if LIST: start.append(LIST[0])
    end.reverse()
    return (start + end)

l = [4,1,99,3,2,2,0,5,6,45,100,34,44,999,998]
print(l)
print(merge_sort(l))
```
> output:
```python
[4, 1, 99, 3, 2, 2, 0, 5, 6, 45, 100, 34, 44, 999, 998]
[0, 1, 2, 2, 3, 4, 5, 6, 34, 44, 45, 99, 100, 998, 999]
```
I'll (_or you can_) merge this once you confirm!

**Comment by christianbender** [comment]
@harshildarji It works for me.

**Pull request merged**